CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 ## 2.4.0 (Unreleased)
 IMPROVEMENTS:
 
-* Change resource handling to use locking mechanism when resource parallel handling is not supported by vCD. [GH-255] 
```

Comment:
I think it wort it to add about test env auto creation

Reply:
done